cmake/Modules/GnuradioConfig.cmake.in

```diff
@@ -17,13 +17,7 @@
 # the Free Software Foundation, Inc., 51 Franklin Street,
 # Boston, MA 02110-1301, USA.
 
-INCLUDE(FindPkgConfig)
-INCLUDE(FindPackageHandleStandardArgs)
-
-# if GR_REQUIRED_COMPONENTS is not defined, it will be set to the following list (all of them)
-if(NOT GR_REQUIRED_COMPONENTS)
-  set(GR_REQUIRED_COMPONENTS RUNTIME ANALOG BLOCKS DIGITAL FFT FILTER PMT)
-endif()
+include(CMakeFindDependencyMacro)
```

Comment:
`find_dependency` apparently needs to include to work. For package config cmake files it's advised to use `find_dependency` instead of `find_package` since it's returns on failure and then stops processing.